src/test/java/com/hotels/beeju/BeejuCoreTest.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (C) 2015-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju;
+
+import com.hotels.beeju.core.BeejuCore;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+
+public class BeejuCoreTest {
```

Comment:
Tests look good :) Think you just need one more test for the third constructor. And also some asserts for all of the options that are being set in 56-78 (like the hive config and driver class name etc)